pkg/internal/cluster/create/actions/config/config.go

```diff
@@ -143,8 +144,11 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 				if err := nodeutils.WriteFile(node, containerdConfigPath, patched); err != nil {
 					return errors.Wrap(err, "failed to write patched containerd config")
 				}
+				if err := waitForSystemdFullyUpAndRunning(node, time.Now().Add(5*time.Second)); err != nil {
+					return errors.Wrap(err, "failed to wait for the systemd to be fully up and running")
+				}
 				// restart containerd now that we've re-configured it
-				if err := node.Command("systemctl", "restart", "containerd").Run(); err != nil {
+				if err := node.Command("bash", "-c", `! systmctl is-system-running || systemctl restart containerd`).Run(); err != nil {
```

Comment:
oops, typoed the first systemctl

we shouldn't need the wait if this is right, though i'm wondering if we should instead structure it like `! containerd-running || restart-containerd`

Reply:
you're right, I've removed the wait. 

I think `! containerd-running || restart-containerd` should work too.